library-sync.sh

```diff
@@ -3,6 +3,71 @@
 # utility script to gather template/imagestream content from https://github.com/openshift/library
 # and store it in this repo (cannot access other repos with dist git, and advised against git submodules
 
+# Without any commandline arguments, this script only updates the OCP supported samples. This behavior
+# can be modified using these commandline arguments:
+# * --okd - the OKD samples are updated
+# * --ocp - the OCP supported samples are updated
+# * --ocp-unsupported - all OCP samples are updated including the unsupported ones
+
+###########################################
+
+# process the commandline args
+PROCESS_OKD="false"
+PROCESS_OCP_UNSUPPORTED="false"
+if [[ $# -eq 0 ]]; then
+  PROCESS_OCP="true"
+else
+  PROCESS_OCP="false"
+fi
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+  --okd)
+    PROCESS_OKD="true"
+    ;;
+  --ocp)
+    PROCESS_OCP="true"
+    ;;
+  --ocp-unsupported)
```

Comment:
minor - I would maybe call this `--ocp-all` as this names might suggest we are updating ONLY the unsopported ones.